bazel/io_opentracing_cpp.patch

```diff
@@ -1,16 +1,36 @@
+diff --git a/mocktracer/BUILD b/mocktracer/BUILD
```

Comment:
Can we get this fixed upstream? CC @g-easy

Reply:
CC @rnburn for OpenTracing ;)

Reply:
https://github.com/opentracing/opentracing-cpp/pull/120